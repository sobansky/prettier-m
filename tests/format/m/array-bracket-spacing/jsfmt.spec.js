run_spec(__dirname, ["babel", "babel-flow", "typescript"], {
    arrayBracketSpacing: true
});