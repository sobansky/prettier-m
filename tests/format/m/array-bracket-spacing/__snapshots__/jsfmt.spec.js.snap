// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`array-bracket-test.js - {"arrayBracketSpacing":true} format 1`] = `
====================================options=====================================
arrayBracketSpacing: true
parsers: ["babel", "babel-flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
const arr = () => {
    const days = ["Mon", "Thue", "Wed"];
}
=====================================output=====================================
const arr = () => {
  const days = [ "Mon", "Thue", "Wed" ];
};

================================================================================
`;
